config/jvm.options

```diff
@@ -16,10 +16,7 @@
 ##
 ################################################################
 
-## GC configuration
-11-13:-XX:+UseConcMarkSweepGC
-11-13:-XX:CMSInitiatingOccupancyFraction=75
-11-13:-XX:+UseCMSInitiatingOccupancyOnly
+17-:-XX:+UseG1GC
```

Comment:
Given this is the default I don't see a reason to be explicit here?

```suggestion
```

Reply:
I thought that being specific was better, in case a future version of JDK move away from G1GC in favor of ZGC, but I'm open to remove it.

Reply:
yeah I don’t think it’s needed, we’ve been surviving without it for a long time ever since we defaulted to jdk17, we typically don't explictly set settings to their defaults in this file. previously we had to because CMS wasn't the default for 11-13